### PR TITLE
Make the TypeScript type a union of the HTML tags

### PR DESCRIPTION
**Pull request by sergeysova**
Relates https://github.com/sergeysova/foliage/issues/2

### CODE REVIEW

**Comment by sindresorhus** [comment]
What's the use-case?

**Comment by sergeysova** [comment]
I want to use exact type as input for fabric-function.

```ts
export function createElement(tag: HTMLTags): Element<HTMLTags> {
```

File: index.d.ts [hunk 1]
```diff
@@ -1,3 +1,122 @@
...
+  | "thead"
+  | "time"
+  | "title"
+  | "tr"
+  | "track"
+  | "u"
+  | "ul"
+  | "var"
+  | "video"
+  | "wbr"
```
Comment (sindresorhus) [inline index.d.ts:118]: Use single-quotes

**Comment by sindresorhus** [comment]
Actually, I just noticed there are more `d.ts` files in the repo. Can you do those too?

**Pull request merged**